.devcontainer/local-features/linux/install.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+echo "Update Apt Database"
+sudo apt-get update
+
+echo "Install Webkit Dependencies"
+sudo apt-get -qq install build-essential libwebkit2gtk-4.0-dev libgtk-3-dev libayatana-appindicator3-dev
```

Comment:
libwebkit2gtk-4.1-dev doesn't seem to be found when trying to install

when running cargo build in devcontainer or locally, get many warnings for name conflicts and some errors such as glib being less than version 2.70. 

will need to touch base with someone about the dependency needs for this. Currently getting info from workflow here: 
https://github.com/DioxusLabs/dioxus/blob/master/.github/workflows/main.yml#L41